app/scripts/modules/core/domain/IComponent.ts

```diff
@@ -0,0 +1,5 @@
+import { IEntityTags } from './IEntityTags';
+
+export interface IComponent {
```

Comment:
This sounds like a member of the `IComponentOptions` or `IComponentController` family. What about `IEntity`?